packages/building-blocks/src/Alert.tsx

```diff
@@ -0,0 +1,124 @@
+import React, { ReactChild, useState } from "react";
+import styled from "styled-components";
+import AnimateHeight from "react-animate-height";
+import { ChevronUp } from "@styled-icons/bootstrap/ChevronUp";
+import { Bell } from "@styled-icons/bootstrap/Bell";
+import blue from "./colors/blue";
+
+interface Props {
+  alertHeader: string;
+  alertSubheader?: string;
+  backgroundColor?: string;
+  collapsable?: boolean;
+  children?: ReactChild;
+  Icon?: any;
```

Comment:
This can probably be a `JSX.Element` or something right?

Reply:
Whoops!! Yes